### ISSUE TITLE
FL0049: Potential false positive with extern/interop methods

### ISSUE THREAD

**Issue by Evangelink** [fsprojects/FSharpLint#487]
### Description

I want to use some C/C++ API from python which contains an underscore in its name. FSharpLint is raising `FL0049` suggesting that I remove the underscore in the name of the function. We can have different names but as far as I know, it's pretty common to keep name "close" to the originals so it might be interesting to either prevent the issue or add some kind of option to decide whether or not to raise in the case of `extern` methods/functions.

### Repro steps

```fsharp
[<DllImport(@"python37", EntryPoint = "PyThreadState_SetAsyncExc", CallingConvention = CallingConvention.Cdecl)>]
extern int PyThreadState_SetAsyncExcLLP64(uint id, IntPtr exc) // FL0049
```

### Expected behavior

No issue raised.

### Actual behavior

An issue is raised.

### Known workarounds

Use the `EntryPoint` part of the `DllImport` attribute.

### Related information

* Operating system: Win 10
* Branch: F# Lint 0.5.2
* .NET Runtime, CoreCLR or Mono Version: .NET 4.8

**Comment by duckmatt** [comment]
Hi @Evangelink, thanks for raising this. I've pushed a fix for this in `0.19.2`

**Comment by knocte** [comment]
Assuming this was fixed in 0.19.2 because nothing was raised here afterwards.
